config/configgrpc/configgrpc_test.go

```diff
@@ -641,6 +641,7 @@ func TestHttpReception(t *testing.T) {
 				assert.NoError(t, errResp)
 				assert.NotNil(t, resp)
 			}
+			assert.NoError(t, grpcClientConn.Close())
```

Comment:
Use defer closer to where the connection is created, since you want that to run even if test stops prematurely.